app/forms/interview_scheduling_form.rb

```diff
@@ -6,7 +6,7 @@ def save
   end
 
   def language_options
-    I18n.backend.translations[I18n.locale][:general][:language].invert
+    I18n.backend.translations[I18n.locale][:general][:language_options].invert
```

Comment:
I updated language list to language_options in order to define a translation for language directly.

Reply:
That's a helpful change